ArcGISRuntimeSDKQt_CppSamples/Maps/ConfigureBasemapStyleLanguage/ConfigureBasemapStyleLanguage.qml

```diff
@@ -0,0 +1,129 @@
+            forceActiveFocus();
+        }
+    }
+
+    // Declare the C++ instance which creates the map etc. and supply the view
+    ConfigureBasemapStyleLanguageSample {
+        id: model
+        mapView: view
+    }
+
+    Rectangle {
+        id: rectangle
+        anchors {
+            right: parent.right
+            top: parent.top
+            margins: 15
+        }
+        width: column.width
+        height: column.height
+        color: Qt.rgba(0, 0, 0, 0.5)
+        border {
+            color: "black"
+            width: 1
+        }
+        opacity: 0.9
+
+        Column {
+            id: column
+            padding: 10
+            spacing: 5
+            width: 300
+
+            Column {
+                id:languageStrategy
+                spacing: 5
+
+                Text {
+                    text: "Set Language Strategy:"
+                    font.bold: true
+                    color: "white"
+                }
+
+                Row {
+                    RadioButton {
+                        id: globalButton
+                        onCheckedChanged: {
+                            model.setNewBasemapLanguage(globalButton.checked, comboBox.currentText);
+                            localButton.checked = !globalButton.checked
+                        }
+                        enabled: comboBox.currentText == "none"
+                    }
+
+                    Text {
+                        id: globalText
+                        text: "Global"
+                        height: globalButton.height
+                        verticalAlignment: Text.AlignVCenter
+                        color: "white"
+                    }
+                }
+
+                Row {
+                    RadioButton {
+                        id: localButton
+                        checked: true
+                        onCheckedChanged: {
+                            model.setNewBasemapLanguage(globalButton.checked, comboBox.currentText);
+                            globalButton.checked = !localButton.checked
+                        }
+                        enabled: comboBox.currentText == "none"
+                    }
+                    Text {
+                        id: localText
+                        text: "Local"
+                        height: localButton.height
+                        verticalAlignment: Text.AlignVCenter
+                        color: "white"
+                    }
+                }
+
+            }
+
+            Column {
+                id:specificLanguage
+
+                Text {
+                    text: "Set Specific Language:"
+                    font.bold: true
+                    color: "white"
+                }
+
+                ComboBox {
+                    id: comboBox
+                    model: ["none" , "Bulgarian", "Greek", "Turkish"]
+                    onCurrentTextChanged: model.setNewBasemapLanguage(globalButton.checked, comboBox.currentText);
```

Comment:
Per our offline discussion, I think this should be more intentional. It seems this executes when the code first loads, but it's unclear from looking at the c++ code how any basemap was loaded. I think the code should be more intentional about how the initial basemap is being set.

When I reviewed the latest code, I assumed no basemap at all was set, because that's what is happening on the C++ side via the constructor.